build_library/grub.cfg

```diff
@@ -6,6 +6,8 @@ set prefix=($root)/flatcar/grub
 # Load any and all video drivers.
 # Required under UEFI to boot Linux with a working console.
 insmod all_video
+# Load the TPM2 module to measure the boot code path and files into PCR 8+9
+insmod tpm
```

Comment:
does it autoload if it's part of CORE_MODULES in `build_library/grub_install.sh`? we need it there for secure boot anyway

Reply:
```
GRUB will not measure its own @file{core.img} - it is expected that firmware
will carry this out. GRUB will also not perform any measurements until the
tpm module is loaded. As such it is recommended that the tpm module be built
into @file{core.img} in order to avoid a potential gap in measurement between
@file{core.img} being loaded and the tpm module being loaded.
```

Reply:
Makes sense, if the module is loaded externally through the config file that's also external it's a chicken-egg problem for the measurement chain :D

Reply:
Seems the config file is part of the efi binary as "memdisk", so that part would be measured. Anyway, added it to the CORE_MODULES and hope that it gets loaded without `insmod` then.

Reply:
It works without `insmod`.